src/Autofac/Core/Activators/Reflection/ReflectionActivator.cs

```diff
@@ -95,7 +134,10 @@ public void ConfigurePipeline(IComponentRegistryServices componentRegistryServic
 
         if (binders.Length == 1)
         {
-            UseSingleConstructorActivation(pipelineBuilder, binders[0]);
+            var singleConstructor = binders[0];
```

Comment:
Does this get us a perf bump I'm not aware of? Or is it just so it's easier to hit a breakpoint and inspect it? (I don't care either way, just didn't know if I was learning something perf-y here.)

Reply:
This was a leftover of a previous set of code where I was capturing the constructor to check for SetsRequiredMembers, I'll change it back.